build.sbt

```diff
@@ -906,6 +936,7 @@ lazy val xlint = Seq(
   scalacOptions += {
     CrossVersion.partialVersion(scalaVersion.value) match {
       case Some((2, scalaMajor)) if scalaMajor >= 12 => "-Xlint:-unused,_"
+      case Some((0, 10))                             => "-Xlint:-unused,_"
```

Comment:
There's no -Xlint in Dotty currently (and I hope scalafix can completely replace -Xlint)